lib/parse.js

```diff
@@ -30,16 +37,27 @@ var parseValues = function parseQueryStringValues(str, options) {
 
         var key, val;
         if (pos === -1) {
-            key = options.decoder(part, defaults.decoder);
+            key = options.decoder(part, defaults.decoder, charset);
             val = options.strictNullHandling ? null : '';
         } else {
-            key = options.decoder(part.slice(0, pos), defaults.decoder);
-            val = options.decoder(part.slice(pos + 1), defaults.decoder);
+            key = options.decoder(part.slice(0, pos), defaults.decoder, charset);
+            val = options.decoder(part.slice(pos + 1), defaults.decoder, charset);
         }
-        if (has.call(obj, key)) {
-            obj[key] = [].concat(obj[key]).concat(val);
+        if (key === 'utf8' && options.utf8Sentinel) {
+            if (val === '✓' || val === '\xe2\x9c\x93') {
```

Comment:
That's a string representation of the 3 octets in the utf-8 encoding of the `✓` character.
> decodeURIComponent('%e2%9c%93')
'✓'

Checking for it here makes sure that we will switch into utf-8 mode when the parameter was parsed as iso-8859-1 because of `defaultCharset`. It's what makes [this test](https://github.com/papandreou/qs/blob/df77dfd7485c5d8ad56cd97e5dd7f5f8168a740a/test/parse.js#L585-L588) pass.

Reply:
Is there a standard for using this character? Should it be customizable, especially if there lacks a standard?

Reply:
See above.